controllers/suite_test.go

```diff
@@ -140,3 +146,163 @@ func randStringRunes(n int) string {
 	}
 	return string(b)
 }
+
+func createNamespace(name string) error {
```

Comment:
Would maybe leave a comment here to remind ourselves that once we switch over to the wrapper from `fluxcd/pkg`, this can be switched to the method the wrapper has to offer. As we tend to forget about these little details.

Reply:
Comment added!